openjpa-persistence-jdbc/src/test/java/org/apache/openjpa/persistence/TestUnwrap.java

```diff
@@ -30,12 +30,22 @@
 
 import org.apache.openjpa.jdbc.conf.JDBCConfiguration;
 import org.apache.openjpa.jdbc.sql.DerbyDictionary;
+import org.apache.openjpa.jdbc.sql.HerdDBDictionary;
 import org.apache.openjpa.kernel.QueryLanguages;
 import org.apache.openjpa.lib.jdbc.DelegatingConnection;
+import org.apache.openjpa.persistence.detachment.model.NoDetachedStateEntityFieldAccess;
+import org.apache.openjpa.persistence.detachment.model.NoDetachedStateEntityPropertyAccess;
 import org.apache.openjpa.persistence.test.SingleEMFTestCase;
 
+import static org.junit.Assume.assumeFalse;
+
 public class TestUnwrap extends SingleEMFTestCase {
 
+    @Override
+    protected void setUp(Object... props) {
+        assumeFalse(this.getDBDictionary() instanceof HerdDBDictionary);
```

Comment:
do you just want to skip this test for HerdDB? In which case you can simply call `setUnsupportedDatabases(HerdDBDictionary.class)`

Reply:
@struberg thanks for the hint.
I wasn't aware of setUnsupportedDatabases 

I will rework this patch